setup/aix61/manualBootstrap.txt

```diff
@@ -28,24 +39,28 @@ http://www.bullfreeware.com/affichage.php?id=2048
 m4-1.4.17-1.aix6.1.ppc.rpm
 sh install-libtool-rpm.sh
 
+4) install ccache
 # install ccache based on these instructions ./setup/ansible-tasks/ccache.yaml
 # we should update that script so that it can be used on AIX as well
 # also currently requires the fix outlined here: https://bugzilla.samba.org/show_bug.cgi?id=11519
 # to be applied manually before the compile
+# install on the new linux machines did not seem to work so use cp and chmod instead
 cd /tmp
 wget http://samba.org/ftp/ccache/ccache-3.2.4.tar.gz
 gunzip ccache-3.2.4.tar.gz
 tar -xvf ccache-3.2.4.tar.gz
 cd ccache-3.2.4
 gmake
-install -c -m 755 ccache /usr/lib
-ln -s /usr/lib/ccache/cc /usr/lib/ccache
-ln -s /usr/lib/ccache/g++ /usr/lib/ccache
-ln -s /usr/lib/ccache/gcc /usr/lib/ccache
+cp ccache /usr/lib
+chmod 755 /usr/lib
+
+ln -s /usr/lib/ccache /usr/lib/cc
+ln -s /usr/lib/ccache /usr/lib/g++
+ln -s /usr/lib/ccache /usr/lib/gcc
```

Comment:
Not sure about base layout on AIX but having these in `/usr/lib` seems a bit off to me. Is `/usr/libexec` a known convention?